Analyzer/src/MakeMiniTree.cc

```diff
@@ -10,110 +10,195 @@ MakeMiniTree::MakeMiniTree()
+            "HBHEIsoNoiseFilter",
+            "HBHENoiseFilter",
         };
-        if( runtype != "MC" ) {
-            variables = {
-            "FinalLumi",
-            "filetag",
-            "NGoodJets_pt30",
-            "NGoodBJets_pt30",
-            "Mbl",
-            "NVtx",
-            "HT_trigger_pt30",
-            "HT",
-            "NGoodElectrons",
-            "NGoodMuons",
-            "MET",
-            };
+
+        if( runtype == "MC" )
+        {
+            variables.insert("Weight");
+            variables.insert("madHT");
+            variables.insert("GenElectrons");
+            variables.insert("GenMuons");
+            variables.insert("GenTaus");
+            variables.insert("GenMET");
+            variables.insert("GenMETPhi");
+            variables.insert("GenParticles");
+            variables.insert("GenParticles_PdgId");
+            variables.insert("GenParticles_ParentId");
+            variables.insert("GenParticles_ParentIdx");
+            variables.insert("GenParticles_Status");
+            variables.insert("ScaleWeights");
+            variables.insert("PSweights");
+            variables.insert("PDFweights");
+            variables.insert("puWeight");
+            variables.insert("puSysUp");
+            variables.insert("puSysDown");
+            variables.insert("NonPrefiringProb");
+            variables.insert("NonPrefiringProbUp");
+            variables.insert("NonPrefiringProbDown");
+            variables.insert("Jets_origIndex");
+            variables.insert("JetsJECup_origIndex");
+            variables.insert("JetsJECdown_origIndex");
+            variables.insert("JetsJERup_origIndex");
+            variables.insert("JetsJERdown_origIndex");
+            variables.insert("Jets_jerFactor");
+            variables.insert("Jets_jecUnc");
+            variables.insert("JetsJECup_jerFactor");
+            variables.insert("JetsJECdown_jerFactor");
+            variables.insert("Jets_jerFactorUp");
+            variables.insert("Jets_jerFactorDown");
+            variables.insert("JetIDJECdown");
+            variables.insert("JetIDJECup");
+            variables.insert("JetIDJERdown");
+            variables.insert("JetIDJERup");
+            variables.insert("JetsAK8_origIndex");
+            variables.insert("JetsAK8JECup_origIndex");
+            variables.insert("JetsAK8JECdown_origIndex");
+            variables.insert("JetsAK8JERup_origIndex");
+            variables.insert("JetsAK8JERdown_origIndex");
+            variables.insert("JetsAK8_jerFactor");
+            variables.insert("JetsAK8_jecUnc");
+            variables.insert("JetsAK8JECup_jerFactor");
+            variables.insert("JetsAK8JECdown_jerFactor");
+            variables.insert("JetsAK8_jerFactorUp");
+            variables.insert("JetsAK8_jerFactorDown");
         }
+
         if( tr.isFirstEvent() ) {
-            std::string myTreeName = "myMiniTree";
+            std::string myTreeName = "SkimmedTree";
```

Comment:
Can we name the tree. The usual tree name? Won't that make using the skims easier?

Reply:
Definitely, works for me. To identify the tree we just change the corresponding "column" in the `sampleSets.cfg` file. Otherwise, I do not think we hardcode the name anywhere.

Reply:
Sounds good, resolved in #91ddf3c